keyboards/pom_keyboards/tnln95/tnln95.c

```diff
@@ -0,0 +1,33 @@
+/* Copyright 2020 Nguyen Minh Hoang
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+#include "tnln95.h"
+
+void keyboard_pre_init_kb(void) {
+    setPinOutput(B1);
+    setPinOutput(B2);
+    setPinOutput(B3);
+
+    keyboard_pre_init_user();
+}
+
+bool led_update_kb(led_t led_state) {
+    bool res = led_update_user(led_state);
+    if(res) {
+        writePin(B1, led_state.num_lock);
+        writePin(B2, led_state.caps_lock);
+    }
+    return res;
+}
```

Comment:
B3 is present in the init, but nothing happens with it after.

Reply:
I commented out B3 (and added a commented E2), I will add functionality for them in the future.